Enhancement - APP - Revisar coherencia entre rangos de fechas y sus etiquetas

jortilles opened the pull request:
## Descripción del Cambio
Se añaden nuevos rangos de tiempo y descripciones a los rangos de tiempo predefinidos para los selectores de fecha.


## Issue(s) resuelto(s)
- solves #193

## Pruebas a realizar para validar el cambio
Hacer un informe con un filtro global con un campo de fecha y comprobar que los nuevos rangos aplican:
* Hoy
* Este mes al completo
* Este año al completo

MarcCornejoJort committed:
DatePicker valores añadidos

jortilles committed:
Mensajes

juanSTIC commented on eda/eda_app/src/app/shared/components/eda-date-picker/eda-date-picker.component.ts:
```diff
@@ -26,18 +26,21 @@ export class EdaDatePickerComponent implements OnChanges {
-		{ label: $localize`:@@DatePickerWeek:Ésta semana`, value: 'weekStart' },
-		{ label: $localize`:@@DatePickerWeekFull:Ésta semana al completo`, value: 'weekStartFull' },
-		{ label: $localize`:@@DatePickerLastWeek:La semana pasada`, value: 'pastWeek' },
+		{ label: $localize`:@@DatePickerWeek:Esta semana (hasta hoy)`, value: 'weekStart' },
```
Este filtro selecciona esta semana hasta el viernes (no hasta hoy)

juanSTIC commented on eda/eda_app/src/app/shared/components/eda-date-picker/eda-date-picker.component.ts:
```diff
@@ -26,18 +26,21 @@ export class EdaDatePickerComponent implements OnChanges {
+		{ label: $localize`:@@DatePickerMonthFull:Este mes completo`, value: 'monthStartFull' },
+		{ label: $localize`:@@DatePickerLastMonth:El mes pasado (hasta equivalente a hoy)`, value: 'pastMonth' },
 		{ label: $localize`:@@DatePickerLastMonthFull:El mes pasado completo`, value: 'pastMonthFull' },
 		{ label: $localize`:@@DatePickerMonthPreviousYear:Éste mes del año pasado`, value: 'monthStartPreviousYear' },
```
El texto debería ser: Este mes del año pasado (hasta equivalente a hoy)

juanSTIC commented on eda/eda_app/src/app/shared/components/eda-date-picker/eda-date-picker.component.ts:
```diff
@@ -26,18 +26,21 @@ export class EdaDatePickerComponent implements OnChanges {
-		{ label: $localize`:@@DatePickerYear:Éste año`, value: 'yearStart' },
+		{ label: $localize`:@@DatePickerYear:Este año (hasta hoy)`, value: 'yearStart' },
+		{ label: $localize`:@@DatePickerYearFull:Este año al completo`, value: 'yearStartFull' },
 		{ label: $localize`:@@DatePickerYearPreviousYear:El año pasado`, value: 'yearStartPreviousYear' },
```
Debería ser: El año pasado (hasta equivalente a hoy)

juanSTIC reviewed (changes requested):
Ver comentarios con inconsistencias.
Además, el filtro "Nulo", es incorrecto, puesto que no filtra las fechas con valor `Null`

juanSTIC commented:
Se acuerda quitar **Fecha nula** del desplegable de rangos

MarcCornejoJort committed:
/**SDA CUSTOM  */ add + elimianación rango null

MarcCornejoJort committed:
/**SDA CUSTOM  */ add + eliminación null rango

MarcCornejoJort committed:
correcciones propuestas

jortilles commented:
Ya se puede volver a probar

juanSTIC reviewed (approved):
Visto y probado

jalbaiges committed:
Update Catalan translations in messages.ca.xlf

jalbaiges committed:
Update English translations for date picker

jalbaiges committed:
Refine Spanish translations in messages.es.xlf

Updated Spanish translations for date picker labels to improve clarity and consistency.

jalbaiges committed:
Refine Spanish translations in messages.gl.xlf

Updated Spanish translations for date picker terms.

jalbaiges reviewed (commented):
Cadenas revisadas. Aprovecho para comentar algunos elementos relativos al proceso de traducción de literales de la interfaz:

1. Idealmente, todos los ficheros de idioma deberían tener la misma estructura, esto es, las mismas cadenas y en las mismas líneas. A modo de ejemplo, en este PR aparece la cadena con id _DatePickerNull_ en español pero no en catalán. Por otro lado, los cuatro ficheros de idioma messages.*.xlf tienen diferente número de líneas.

2. Al revisar las cadenas propias del PR se ha observado que en el fichero de catalán hay cadenas con errores ortográficos o que no siguen los estándares a aplicar en interfaces de usuario. Dado que tiempo atrás se realizó una revisión completa del fichero es de suponer que a) se han ido añadiendo cadenas para las que no se ha solicitado revisión o b) se han sobreescrito (las) correcciones anteriores. Es MUY importante asegurar que el despliegue de nuevas cadenas SIEMPRE pasa el pertinente proceso de revisión y que en ningún caso las cadenas revisadas puedan ser revertidas a valores anteriores.

3. No es inusual que aparezcan cadenas con espacios en blanco antes y/o después del literal. Asumiendo que esos espacios no son relevantes, debería asegurarse que no se incluyen en los ficheros a fin de evitar su eliminación manual.

4. Lo mismo sucede con líneas cortadas aparentemente de forma innecesaria; con los niveles de indentación que varían a lo largo del fichero, etc. Por lo general sería conveniente que la estructura y el contenido de los ficheros fuera más consistente/estandarizada.

PaulaaSTIC reviewed (approved):
(A)probado
